gauntlet/CHANGELOG.md

```diff
@@ -7,44 +7,48 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
-#### Added
+### Added
```

Comment:
@a-frantz - Didn't you recently add the extra layer of indentation to this file?